### DISCUSSION TITLE
Question about `suggestion.auto_trigger`

### DISCUSSION THREAD

**Discussion by AnsonH** [zbirenbaum/copilot.lua#92] [Q&A] [answered]
# Question

According to the [suggestion section of README](https://github.com/zbirenbaum/copilot.lua#suggestion):
> When `auto_trigger` is `true`, copilot starts suggesting as soon as you enter insert mode.
> To toggle auto trigger for the current buffer, use `require("copilot.suggestion").toggle_auto_trigger()`.

Does that mean if I set `suggestion.auto_trigger` to `true`, Copilot should generate **inline suggestions** as I type in insert mode as demonstrated in the below screenshot?

![image](https://user-images.githubusercontent.com/57580593/206828426-e22d1805-c6f4-4568-9958-30b2ccea45c0.png)

**Comment by AnsonH** [comment, marked as answer]
Update: I figured out the answer by myself. The answer is **YES**.

Originally I'm asking this question because Copilot is not generating any inline suggestions as I type in insert mode. Apparently, I have to be patient and wait for 10~15 seconds in insert mode before Copilot will start generating inline suggestions 😅.

**Comment by melonges** [comment]
hey! copilot dont work in lunarvim
I inserted this code than success logged in, but copilot stil dont work
https://www.lunarvim.org/docs/plugins/extra-plugins#copilotlua-and-copilot-cmp

**Reply by MunifTanjim** [reply]
That documentation looks outdated. Try doing this:

```lua
lvim.plugins = {
  { "zbirenbaum/copilot.lua",
    event = { "VimEnter" },
    config = function()
      require("copilot").setup({
        suggestion = {
          auto_trigger = true,
          keymap = {
            -- change these if you want
            accept = "<M-l>",
            next = "<M-]>",
            prev = "<M-[>",
            dismiss = "<C-]>",
          },
        },
      })
    end,
  },

  -- other plugins
}
```

After that you can run `:Copilot status` to see if it's running.

And use the keymaps to use copilot.
